server/setup.go

```diff
@@ -43,18 +44,27 @@ func Setup() error {
 	loadConfig()
 	log.Printf("Using %s for persistence\n", *database)
 
-	ctx := context.Background()
+	ctx, fn := context.WithTimeout(context.Background(), 4*time.Minute)
+	defer fn()
+
+	log.Printf("Fetching DB (createTables: %v)\n", *createTables)
```

Comment:
IIRC I added these when I had no clue what I was doing and trying to get connected to RDS. Maybe we can clean these up?